Exiled.API/Extensions/UserGroupExtensions.cs

```diff
@@ -38,5 +38,15 @@ public static bool EqualsTo(this UserGroup @this, UserGroup other)
         /// <returns>The key of that group, or null if not found.</returns>
         public static string GetKey(this UserGroup @this) => Server.PermissionsHandler._groups
             .FirstOrDefault(pair => pair.Value.EqualsTo(@this)).Key;
+
+        public static UserGroup GetValue(string GroupName)
+        {
+            var groups = ServerStatic.GetPermissionsHandler().GetAllGroups();
+            if (groups.ContainsKey(GroupName))
```

Comment:
Can't you directly use `ServerStatic.GetPermissionsHandler().GetAllGroups().TryGetValue(...)`?